### DISCUSSION TITLE
Pagination in getStaticPaths

### DISCUSSION THREAD

**Discussion by AlexanderBachSchmidt** [vercel/next.js#18810] [Help]
Hi all,

I'm using Prismic and need to generate static paths for all my products. Since I have more than 20, I can't receive them all in one request.

What is the best way to implement pagination into the gatStaticPaths call, so that I can get all products? I have not been able to find any examples at all of this.

Thank you

**Comment by joshuaoas** [comment]
The closest thing to an answer on this is <a href="https://github.com/vercel/next.js/discussions/10987">Here</a> Unfortunately I was never able to get this to work, and this feature isn't in any example.
